src/frontend/apps/web/index.html

```
@@ -7,7 +7,8 @@
     <title>Warren</title>
   </head>
   <body>
-    <div id="root"></div>
+    <div id="warren-frontend-data" data-context="{}"></div>
```

FMI, in which case is this file used ? And how data-context will be populated ?

---

In terms of developer experience, I plan to add a way to mock data context while running the `vite dev` server. Hot reloading is much faster, and sometimes errors a bit more explicit. I removed the command from the package.json, to avoid any confusion. Do you want me to remove this file?

I haven't yet designed the "development" mode relying only on the frontend server, which would bypass the Django server. I'll keep you posted.